types/react-drag-listview/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for react-drag-listview 0.1
+// Project: https://github.com/raisezhang/react-drag-listview
+// Definitions by: hirohe <https://github.com/hirohe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+
+export interface DragListViewProps {
+    // on drag end callback, required
+    onDragEnd: (fromIndex: number, toIndex: number) => void;
+    // get drag handle cssQuery
+    handleSelector?: string;
+    // get drag item cssQuery
+    nodeSelector?: string;
+    // ignore node list
+    ignoreSelector?: string;
+    // whether use auto scroll for dragging
+    enableScroll?: boolean;
+    // scroll speed
+    scrollSpeed?: number;
+    // get dragLine's className, css properties must be use !important
+    lineClassName?: string;
```

Comment:
```suggestion
    // get drag handle cssQuery
    handleSelector?: string | undefined;
    // get drag item cssQuery
    nodeSelector?: string | undefined;
    // ignore node list
    ignoreSelector?: string | undefined;
    // whether use auto scroll for dragging
    enableScroll?: boolean | undefined;
    // scroll speed
    scrollSpeed?: number | undefined;
    // get dragLine's className, css properties must be use !important
    lineClassName?: string | undefined;
```
you can probably use strict optional prop types compatiblity here, those are props checked by expressions like `this.props.scrollSpeed` in the source:
https://github.com/raisezhang/react-drag-listview/blob/cd0dcfa122b82893491a7b1168ab3d2675aac173/src/ReactDragListView.jsx
this would work nice if someone is using TS4.4 strict prop type checks